doc/calculator/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "calculator"
-version = "0.20.0"
+version = "0.20.1"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 workspace = "../.." # <-- We added this and everything after!
 edition = "2021"
 
 [build-dependencies]
-lalrpop = { version = "0.20.0", path = "../../lalrpop" }
+lalrpop = { version = "0.20.1", path = "../../lalrpop" }
 
 [dependencies]
-lalrpop-util = { version = "0.20.0", path = "../../lalrpop-util", features = ["lexer", "unicode"] }
+lalrpop-util = { version = "0.20.1", path = "../../lalrpop-util", features = ["lexer", "unicode"] }
```

Comment:
Similarly, can we do something like:

```
lalrpop-util = { workspace = true, features = [ "lexer", "unicode" ] }
```

here, and put in the main `Cargo.toml` something like:

```
lalrpop-util = { version = "0.20.1", path = "./lalrpop-util"}
```

Sorry, it might be out of scope of this PR, but I've been doing that in other projects (define workspace-level dependencies for workspace members, and a workspace-level version, and use that everywhere else so that you only have to bump each member's, and only if it doesn't inherit the workspace's main version) and I wonder if it could make the workflow simpler for LALRPOP. Feel free to tell me that we can see that later :slightly_smiling_face:

Reply:
See #843